php-checks/src/main/java/org/sonar/php/checks/formatting/TokenUtils.java

```diff
@@ -20,16 +20,12 @@
 package org.sonar.php.checks.formatting;
 
 import com.google.common.base.Preconditions;
-import com.sonar.sslr.api.AstNode;
-import com.sonar.sslr.api.Token;
 import com.sonar.sslr.api.TokenType;
-import org.sonar.php.checks.FormattingStandardCheck;
+import org.sonar.plugins.php.api.tree.lexical.SyntaxToken;
 
-public abstract class SpacingCheck {
+public class TokenUtils {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)